packages/suite/src/components/suite/modals/ReduxModal/ConfirmValueModal/ConfirmValueModal.tsx

```diff
@@ -135,6 +133,9 @@ export const ConfirmValueModal = ({
         }
     }, [canConfirmOnDevice, dispatch, isConfirmed, modalContext, validateOnDevice]);
 
+    // light theme colors are used always for QR code
+    const qrCodeFgColor = addressConfirmed ? colorVariants.standard.iconDefault : colorVariants.standard.iconSubdued
```

Comment:
what about using color from `palette` directly? 

e.g.`palette.darkGray1000`?

Reply:
Done (though `lightGray1000` is the appropriate one)